web/src/main/java/org/apache/shiro/web/util/WebUtils.java

```diff
@@ -63,6 +63,15 @@
     public static final String SAVED_REQUEST_KEY = "shiroSavedRequest";
 
     /**
+	 * If the {@link WebEnvironment} was instantiated within
+	 * {@link #getRequiredWebEnvironment(ServletContext, boolean)} this
+	 * attribute contains the respective {@link EnvironmentLoader}
+	 * 
+	 * @since 1.4
```

Comment:
the `@since` tags would be at least would need to be `1.5` as this would be a new feature